experimental/modular_oak_functions/src/log.rs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::{Service, ServiceProxy};
+
+/// Service to provide loggin functionality.
```

Comment:
Will we have one logger or many? Who will be using the logger?

Reply:
There will be one logging service implementation per runtime. This logging service will be used primarily by the Wasm workload to do debug logging.